src/Usb.Net/Windows/WindowsUsbInterface.cs

```diff
@@ -70,6 +70,27 @@ public void Dispose()
 
             GC.SuppressFinalize(this);
         }
+
+        public Task<ReadResult> SendControlTransferAsync(SetupPacket setupPacket, byte[] buffer, CancellationToken cancellationToken = default)
+        {
+            return buffer == null || buffer.Length == 0
```

Comment:
@MelbourneDeveloper I think passing null buffers should be allowed there, use case on STM32 is when sending a "Clear Status" or "Reboot" command.

Reply:
The plan is to remove the buffer parameter entirely. The result will get returned.